ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard.component.ts

```diff
@@ -105,7 +105,7 @@ export class DashboardComponent implements OnInit, OnDestroy {
 
     public ngOnInit(): void {
         this.studyGuid = this.toolkitConfiguration.studyGuid;
-        const anno = this.announcements.getMessage(this.studyGuid)
+        const anno = this.announcements.getMessages(this.studyGuid)
             .pipe(
                 filter(x => x !== null && x.length !== 0),
```

Comment:
Might be simpler:
```suggestion
                filter(messages => !!messages),
```
Also consider just using an `async` pipe from template to get announcements.

Reply:
Thanks for the `filter` changes suggestion.

I also thought about `async` pipe, but we should change messages state:
``` typescript
    public closeMessage(index: number): void {
        this.announcementMessages[index].shown = false;
    }
```
`async` pipe will bring unnecessary complexity.